apps/api/src/main.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * This is not a production server yet!
+ * This is only a minimal backend to get started.
+ */
+
+import { Logger } from '@nestjs/common';
+import { NestFactory } from '@nestjs/core';
+
+import { AppModule } from './app/app.module';
+
+async function bootstrap() {
+  const app = await NestFactory.create(AppModule);
+  const globalPrefix = 'api';
+  app.setGlobalPrefix(globalPrefix);
+  const port = process.env.PORT || 5000;
```

Comment:
Maybe we can move `process.env` and other global variables to .env file or smth?

Reply:
Later we will create `.env` file for db config and others